src/code-util.jl

```diff
@@ -57,6 +57,21 @@ function convert_input_dict_to_string(inputs::AbstractDict)
     return program
 end
 
+function convert_install_dict_to_string(inputs::AbstractDict{String, String})
```

Comment:
Will this be readonly? This will break query layering if it's not a write query

Reply:
It is not readonly. The query is run with `_execute_test` (to get all the fancy error handling) with `readonly=false`.

Reply:
nice!!